airbyte-cdk/java/airbyte-cdk/core/src/testFixtures/kotlin/io/airbyte/cdk/integrations/base/ssh/SshBastionContainer.kt

```diff
@@ -144,13 +144,8 @@ class SshBastionContainer : AutoCloseable {
          */
         fun getInnerContainerAddress(container: Container<*>): ImmutablePair<String, Int> {
             return ImmutablePair.of(
-                container.containerInfo.networkSettings.networks.entries
-                    .stream()
-                    .findFirst()
```

Comment:
findFirst().get().value would throw an exception if the stream is empty. first() throws an exception if the stream is empty.